docs/extensions/index.md

```diff
@@ -222,6 +222,18 @@ When a user installs this extension, they will be prompted to enter their API
 key. The value will be saved to a `.env` file in the extension's directory
 (e.g., `<home>/.gemini/extensions/my-api-extension/.env`).
 
+You can view a list of an extension's settings by running:
+
+```
+gemini extensions configure list <extension name>
```

Comment:
nit: why `configure` versus `settings` as the name? I think settings would be more consistent with the `/settings` command. To me, `configure list` is a really odd combination of words, am I configuring the list of extensions?